### PR TITLE
Create unshorten module

### PR DESCRIPTION
**Pull request by Abdiramen**
Implements unshortening from issue #29.

### CODE REVIEW

**Comment by LinuxMercedes** [comment]
I'm just checking from my phone so forgive me if this is obvious, but what
is file.txt for?

File: modules/unshorten/file.txt [hunk 1]
```diff
@@ -0,0 +1,904 @@
+IncomingMessage {
```
Comment (euank) [inline modules/unshorten/file.txt]: Accidental dev/debug file?

File: modules/unshorten/file.txt [hunk 2]
```diff
@@ -0,0 +1,904 @@
...
+     'Connection',
+     'Transfer-Encoding',
+     'Set-Cookie',
+     'JSESSIONID=6E945BB36963A1C297F47E3E47190ACC; Path=/; HttpOnly',
+     'Set-Cookie',
+     'ebay=%5Esbf%3D%23%5E;Domain=.ebay.com;Path=/ ',
+     'Set-Cookie',
+     'dp1=bu1p/QEBfX0BAX19AQA**5cf2b88f^bl/US5ed3ec0f^;Domain=.ebay.com;Expires=Sun, 31-May-2020 17:40:31 GMT;Path=/ ',
+     'Set-Cookie',
+     's=CgAD4ACBbEtaPYmM2ZmM0NzkxNjMwYWFkY2Y0NjIwN2E3ZmZkYWI4NWWVtK8Q;Domain=.ebay.com;Path=/; HttpOnly ',
```
Comment (euank) [inline modules/unshorten/file.txt]: Note, you might want to reset your password or logout/login if this is from a logged in session; I'm not checking the cookie to see if it's valid, but either way it's out there now.

File: modules/unshorten/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      reply(`${response.request.uri.href}`);
+    } else {
+      reply(`${link}: redirect fails`);
+    }
+  });
+}
+
+
+function usage(command, reply) {
+  reply(`${command} ${use[command]}`);
```
Comment (euank) [inline modules/unshorten/index.js]: This should be one reply with a newline in the text; In this case it's no different, but in general reply has internal logic to handle multi-line content, and well behaved modules shouldn't avoid that without good reason.

File: modules/unshorten/file.txt (hunk 1 above)
Comment (Abdiramen) [reply modules/unshorten/file.txt]: Yikes, didn't realize I added this :grimacing:

File: modules/unshorten/file.txt (hunk 2 above)
Comment (Abdiramen) [reply modules/unshorten/file.txt]: Thanks for the heads up!

File: modules/unshorten/index.js (hunk 3 above)
Comment (Abdiramen) [reply modules/unshorten/index.js]: They should be joined now.

**Comment by euank** [comment]
Code-wise LGTM.

One last nit is that I think the commits should be squashed (no need for file.txt to be in our git log), and I've been using a 'subsystem' style commit message format generally, so e.g. the commit message here would be prefixed with 'modules/unshorten'

**Comment by Abdiramen** [comment]
@euank took a second to figure out how to remove commits but I think everything should to satisfactory now.

**Pull request merged**